commands/roles/removeRole.js

```diff
@@ -0,0 +1,41 @@
+const DiscordUtil = require('../../common/discordutil.js');
```

Comment:
I think @marg30 updated this command, but I quickly spun this up to get the bot to start.